images/macos/software-report/SoftwareReport.Generator.ps1

```diff
@@ -53,187 +42,123 @@ if ($os.IsLessThanBigSur) {
+    (Get-PackerVersion),
+    (Get-OpenSSLVersion),
+    (Get-JqVersion),
+    (Get-GPGVersion),
+    (Get-PostgresClientVersion),
+    (Get-PostgresServerVersion),
+    (Get-Aria2Version),
+    (Get-AzcopyVersion),
+    (Get-ZstdVersion),
+    (Get-BazelVersion),
+    (Get-BazeliskVersion),
+    (Get-HelmVersion),
+    (Get-MongoVersion),
+    (Get-MongodVersion),
+    (Get-7zipVersion),
+    (Get-BsdtarVersion),
+    (Get-GnuTarVersion)
 )
 if ($os.IsHigherThanMojave) {
-    $newmanVersion = Run-Command "newman --version"
-    $markdown += New-MDList -Lines "Newman $newmanVersion" -Style Unordered -NoNewLine
+    $markdown += New-MDList -Lines (Get-NewmanVersion) -Style Unordered -NoNewLine
 }
 if ($os.IsLessThanBigSur) {
-    $vagrant = Run-Command "vagrant -v"
-    $vbox = Run-Command "vboxmanage -v"
-    $parallelVersion = Run-Command "parallel --version" | Select-String "GNU parallel" | Select-Object -First 1
     $markdown += New-MDList -Style Unordered -Lines @(
-        "virtualbox $vbox",
-        $vagrant,
-        $parallelVersion
+        (Get-VirtualBoxVersion),
+        (Get-VagrantVersion),
+        (Get-ParallelVersion)
     )
 }
 $markdown += New-MDNewLine
 
 # Tools
-$fastlaneVersion = Run-Command "fastlane --version" | Select-String "^fastlane [0-9]" | Take-Part -Part 1
-$cmakeVersion = Run-Command "cmake --version" | Select-Object -First 1 | Take-Part -Part 2
-$appcenterCLIVersion = Run-Command "appcenter --version" | Take-Part -Part 2
-$azureCLIVersion = Run-Command "az -v" | Select-String "^azure-cli" | Take-Part -Part 1
-$awsVersion = Run-Command "aws --version" | Take-Part -Part 0 | Take-Part -Delimiter "/" -Part 1
-$aliyunVersion = Run-Command "aliyun --version" | Select-String "Alibaba Cloud Command Line Interface Version " | Take-Part -Part 6
-$awsSamVersion = Run-Command "sam --version" | Take-Part -Part 3
-$awsSessionManagerVersion = Run-Command "session-manager-plugin --version"
-$ghcUpVersion = Run-Command "ghcup --version" | Take-Part -Part 5
-$ghcVersion = Run-Command "ghc --version" | Take-Part -Part 7
-$cabalVersion = Run-Command "cabal --version" | Take-Part -Part 3
-$stackVersion = Run-Command "stack --version" | Take-Part -Part 1 | ForEach-Object {$_.replace(",","")}
-$xcodeVersion = Run-Command "pkgutil --pkg-info com.apple.pkg.CLTools_Executables" | Select -Index 1 | Take-Part -Part 1
-
 $markdown += New-MDHeader "Tools" -Level 3
 $markdown += New-MDList -Style Unordered -NoNewLine -Lines @(
-    "Fastlane ${fastlaneVersion}",
-    "Cmake ${cmakeVersion}",
-    "App Center CLI ${appcenterCLIVersion}",
-    "Azure CLI ${azureCLIVersion}",
-    "AWS CLI ${awsVersion}",
-    "AWS SAM CLI ${awsSamVersion}",
-    "AWS Session Manager CLI ${awsSessionManagerVersion}",
-    "Aliyun CLI ${aliyunVersion}",
-    "Xcode Command Line Tools ${xcodeVersion}"
+    (Get-FastlaneVersion),
+    (Get-CmakeVersion),
+    (Get-AppCenterCLIVersion),
+    (Get-AzureCLIVersion),
+    (Get-AWSVersion),
+    (Get-AWSSAMVersion),

```

Comment:
nitpick: `AWSCLIVersion` and `AWSSAMCLIVersion` for consistency with other tools naming

Reply:
Renamed these functions